changelog/next/bug-fixes/4933--parsing_functions.md

```diff
@@ -0,0 +1,3 @@
+Re-defining a predefined GROK pattern no longer terminates the application.
+
+The `string.parse_json()` function can now parse scalars as intended.
```

Comment:
This bug fix description is hard to understand for users. What was broken before?